### PR TITLE
My Portfolio Project

### PR DESCRIPTION
**Pull request by efoosag**
This is the front page of my portfolio where I carried the following task:
- Create repository withe readme file
- Create Branch
- Applying best practices for HTML and CSS
- Applying Github Flow
- Descriptive and Understanding Commit Messages and ReadMe file

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  margin: 0 24px 0 24px;
+  padding-top: 96px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
```
Comment (Omar-Muhamad) [inline style.css:68]: - [ ] I can see that your main header overlap with the background image please fix that and try to stick as much as possible to the given Figma design.
[hint] you can do it by adjusting your header width.

| Figma Design  | Your Design |
| ------------- | ------------- |
|image  | image  |

Comment (efoosag) [reply style.css:68]: I think it is as a result of the font-family. Sans-serif instead of Poppins

File: style.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 78px;
+  margin: 0 24px;
+}
+
+.logo {
+  font-family: 'Poppins', sans-serif;
```
Comment (AnselemOdims) [inline style.css:36]: - [ ] [OPTIONAL] At the moment, it is `san-serif` font that is displaying. You should use the `Poppins` which can be added by importing from google fonts

**Review by AnselemOdims — APPROVED** [review]
### Hi  @efoosag 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**